mmv1/products/secretmanager/product.yaml

```diff
@@ -14,6 +14,7 @@
 --- !ruby/object:Api::Product
 name: SecretManager
 display_name: Secret Manager
+legacy_name: secret_manager
```

Comment:
I think we can omit this from the existing service, since its name can't be changed in any case.